playground/src/share-view/share-view.js

```diff
@@ -0,0 +1,75 @@
+// Copyright 2020 The AMPHTML Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import './share-view.scss';
+import template from './share-view.hbs';
+
+import {Base64} from 'js-base64';
+import FlyIn from '../fly-in/fly-in.js';
+import * as Button from '../button/button.js';
+
+export function createShareView(editor) {
+  const target = document.getElementById('share-view');
+  const trigger = document.getElementById('share-toggle');
+  if (!target) return;
```

Comment:
Couldn't you exit early? If there's no toggle there's no need to initialize the view, is there?